game/src/Game/Autoload/Events.gd

```diff
@@ -19,9 +19,10 @@ var _define_filepaths_dict : Dictionary = {
 
 # REQUIREMENTS
 # * FS-333, FS-334, FS-335, FS-341
-func _ready():
+func load_events():
```

Comment:
```suggestion
func load_events(loading_screen : LoadingScreen):
```
Better to avoid hardcoded nodepaths, replace `$/root/GameStartup/LoadingScreen` with `loading_screen` below, maybe even make them `if loading_screen: loading_screen.update_loading_screen(...)` so that loading could still happen without the screen.